JitsiMeetJS.js

```diff
@@ -42,6 +43,9 @@ var LibJitsiMeet = {
 
     version: '{#COMMIT_HASH#}',
 
+    // expose JitsiTrackError this way to give library consumers to do checks
+    // like if (error instanceof JitsiMeetJS.JitsiTrackError) { }
+    JitsiTrackError: JitsiTrackError,
```

Comment:
JitsiTrackError is constructor, JitsiTrackErrors is enum.

We already expose JitsiConnection, JitsiConnectionErrors and JitsiConnectionEvents the same way: https://github.com/jitsi/lib-jitsi-meet/blob/master/JitsiMeetJS.js#L45, https://github.com/jitsi/lib-jitsi-meet/blob/master/JitsiMeetJS.js#L54, https://github.com/jitsi/lib-jitsi-meet/blob/master/JitsiMeetJS.js#L48

I can suggest to rename JitsiTrackErrors to JitsiTrackErrorNames or JitsiTrackErrorsEnum. But we still need to expose JitsiTrackError constructor to give library users opportunity to do checks like:
```javascript
if (error instanceof JitsiMeetJS.JitsiTrackError) {
    switch (error.name) {
        case JitsiMeetJS.errors.track.PERMISSION_DENIED:
            // do something
    }
}
```


So I can suggest renaming